src/lua/CommunityBalanceMod/FileHooks.lua

```diff
@@ -119,6 +119,7 @@ Advanced Support Name,
 FortressPvE
 ==]]
 Loader_SetupFilehook("lua/TechData.lua", "post", folder)
+Loader_SetupFilehook("lua/TeamInfo.lua", "post", folder)
```

Comment:
/Combined/TeamInfo.lua is already loaded at line 97

Reply:
Removed duplicate